pkg/codegen/testing/test/program_driver.go

```diff
@@ -375,6 +375,11 @@ var PulumiPulumiProgramTests = []ProgramTest{
 		Description: "Testing for expressions with typed target expressions in csharp",
 		Skip:        allProgLanguages.Except("dotnet"),
 	},
+	{
+		Directory:   "csharp-empty-list-property",
```

Comment:
This feels like a useful test to have for all the languages? Can we try testing all, and if any others don't work skip them for now but raise issues for them.

Also rename the test to just "empty-list-property"

Reply:
✅